result/block-scoped-var.js

```diff
@@ -0,0 +1,57 @@
+/*eslint block-scoped-var: "error"*/
+
+function doIf() {
```

Comment:
__[development][eslint]__
* Error - 'doIf' is defined but never used (no-unused-vars)